package.json

```diff
@@ -23,7 +23,7 @@
     "deepcopy": "1.0.0"
   },
   "peerDependencies": {
-    "webpack": "4.28.4"
+    "webpack": "^4.28.4"
```

Comment:
should we change this to ^4.0.0?

Reply:
If we know for sure it works with older versions. Your developing with this version I think it's safer to don't allow older versions. Also it should be easy for someone to upgrade because there shouldn't be any breaking changes in minor updates.